src/extension/background.js

```diff
@@ -539,6 +539,24 @@ const externalActions = {
     }
     return resp;
   },
+  // close palette with given id
+  closePalette: async ({ id }, { tab }) => {
+    try {
+      await chrome.scripting.executeScript({
+        target: { tabId: tab.id },
+        args: [id],
+        func: (paletteId) => {
+          const palette = document.querySelector('helix-sidekick')
+            ?.shadowRoot?.getElementById(`hlx-sk-palette-${paletteId}`);
```

Comment:
This should probably be changed to use `window.hlx.sidekick`, and avoid using optional chaining (older Safaris lacking support for it are still around):
```suggestion
          const palette = window.hlx && window.hlx.sidekick
            && window.hlx.sidekick.shadowRoot.getElementById(`hlx-sk-palette-${paletteId}`);
```